nncf/common/pruning/statistics.py

```diff
@@ -133,3 +133,55 @@ def to_str(self) -> str:
             f'Statistics of the filter pruning algorithm:\n{algorithm_string}'
         )
         return pretty_string
+
+
+class PrunedModelTheoreticalBorderline(Statistics):
+    """
+    Contains theoretical borderline statistics of the filter pruning algorithm.
+    """
+
+    def __init__(self,
+                 pruned_layers_num: int,
+                 prunable_layers_num: int,
+                 minimum_possible_flops: float,
+                 minimum_possible_params: float,
+                 full_flops: int,
```

Comment:
what are the `full_flops` and `full_params_num`, no docstrings are provided?

Reply:
Thank you for your vigilance! Done

Reply:
Welcome! But the description you provided does not differ from the name of the variable)

Can you please change it to `full_flops: the total amount of FLOPS in the model`?

Reply:
Sure, I've renamed this parameters for FilterPruningStatistics as well

Reply:
You could also make them more meaningful by renaming to total_flops and total_params.

Reply:
I've change the name only in `PrunedModelTheoreticalBorderline` class and haven't changed it in `FilterPruningStatistics` because `full_flops` name is well-established in nncf code. For example: https://github.com/openvinotoolkit/nncf/blob/develop/nncf/torch/pruning/filter_pruning/global_ranking/evolutionary_optimization.py#L310

If you really would like to rename all this `full_flops` I'll do it in a moment

Reply:
Let's only change the new code and don't just rename :)